docs/pages/versions/unversioned/sdk/sensors.mdx

```diff
@@ -6,18 +6,23 @@ packageName: 'expo-sensors'
 
 import { APIInstallSection } from '~/components/plugins/InstallSection';
 import PlatformsSection from '~/components/plugins/PlatformsSection';
+import { BoxLink } from '~/ui/components/BoxLink';
 
-**`expo-sensors`** provides various APIs for accessing device sensors to measure motion, orientation, pressure, magnetic fields, and step count.
+**`expo-sensors`** provides various APIs for accessing device sensors to measure motion, orientation, pressure, magnetic fields, ambient light, and step count.
```

Comment:
```suggestion
`expo-sensors` provides various APIs for accessing device sensors to measure motion, orientation, pressure, magnetic fields, ambient light, and step count.
```

Unrelated to this PR: the `**` outside of backticks doesn't bold the text when markdown is rendered on a page. I'd suggest either using `**expo-sensors**` or removing `**`.